helpers/framed.js

```diff
@@ -21,7 +21,7 @@ export async function isFramed() {
 	framed = await Promise.race([
 		new Promise(resolve => {
 			const handleIsFramedResponse = evt => {
-				if (!evt || !evt.data || !evt.data.isFramed === undefined) return;
+				if (!evt || !evt.data || evt.data.isFramed === undefined) return;
```

Comment:
Ran into this while testing. Accidentally inverted this logic when I switched this from `!evt.data.isFramed` to an explicit `undefined` check. 😬

Reply:
Maybe this was intended to be `if (!evt || !evt.data || !evt.data.isFramed)`?

Reply:
Oh, that's what it used to be.  Was the explicit check needed?

Reply:
No, it just felt... More correct? The host page will only ever return `true` or `undefined`, but it still feels more correct to explicitly resolve to `false` if `isFramed` happens to be false somehow.

Reply:
Sometimes I write it to be specific, but other times not, to be more tolerant of both `undefined` and `null` possible values.  It's not an issue as long a `null` is never a possible value, so more just wondering.

Reply:
Fair enough. Yeah, this was a bit of a weird one.